src/TinyUPnP.cpp

```diff
@@ -697,25 +721,30 @@ boolean TinyUPnP::getIGDEventURLs(gatewayInfo *deviceInfo) {
 				urlBaseFound = true;
 			}
 		}
+
+		// to support multiple <serviceType> tags
```

Comment:
@GIPdA can you please explain why you needed this?

Reply:
➡️ Moving over to https://github.com/ofekp/TinyUPnP/pull/42 instead since it has a branch.